app/views/admin/promotional_feature_items/_promotional_feature_item.html.erb

```diff
@@ -22,9 +22,5 @@
   </p>
 
   <%= link_to "Edit", edit_admin_organisation_promotional_feature_item_path(@organisation, @promotional_feature, promotional_feature_item), class: "btn btn-default"%>
-  <%= button_to "Delete",
-        admin_organisation_promotional_feature_item_path(@organisation, @promotional_feature, promotional_feature_item),
-        method: "delete",
-        class: "btn btn-danger",
-        data: { confirm: "Are you sure you want to delete this feature item?" } %>
+  <%= link_to "Delete", confirm_destroy_admin_organisation_promotional_feature_item_path(@organisation,@promotional_feature, promotional_feature_item), class: 'btn btn-danger btn-sm' %>
```

Comment:
I thin for now we should leave this as a button, and update it to a link in the GOV.UK Design System implementation

```
<%= button_to "Delete",
        confirm_destroy_admin_organisation_promotional_feature_item_path(@organisation,@promotional_feature, promotional_feature_item),
        method: "get",
        class: "btn btn-danger",
```